app/controllers/concerns/api_pagination.rb

```diff
@@ -7,7 +7,7 @@ module ApiPagination
 private
 
   def paginate(scope)
-    _pagy, paginated_records = pagy(scope, items: per_page, page:)
+    _pagy, paginated_records = pagy(scope, limit: per_page, page:)
```

Comment:
@cwrw I suspect you'll want to double check this and click the button so feel free 🙂